components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabIntentProcessor.kt

```diff
@@ -22,12 +21,11 @@ import mozilla.components.support.utils.SafeIntent
 class CustomTabIntentProcessor(
     private val sessionManager: SessionManager,
     private val loadUrlUseCase: SessionUseCases.DefaultLoadUrlUseCase,
-    private val displayMetrics: DisplayMetrics
+    private val resources: Resources
 ) : IntentProcessor {
 
     override fun matches(intent: Intent): Boolean {
-        val safeIntent = SafeIntent(intent)
```

Comment:
We should keep this since activities can make direct calls to `matches` and we should use a `SafeIntent` for that.